base.lua

```diff
@@ -585,6 +585,16 @@ function courseplay:onUpdate(dt)
 		self.cp.infoText = nil
 	end
 
+	if self.cp.firstRun == nil then 
+		if self.cp.driver then 
+			---Post init function, as not all giants variables are
+			---set correctly at the first courseplay:setAIDriver() call.
+			self.cp.driver:postInit()
+			self.cp.firstRun = true
```

Comment:
This is confusing, if firstRun is true, I'd think it is the first run, but the opposite seems to be the case. I'd recommend renaming it to `postInitDone` or something more descriptive.